### PR TITLE
Issue 230 - Show system time on front page

### LINKED ISSUE #230: Nice to have: Show system time on front page
Would be nice to see what time the system things it is somewhere on the front page of scale.

### PR DESCRIPTION
**Pull request by Nikkala-AIS**
Added a simple system clock to the navbar for screen sizes over 1350px wide (avoiding conflict with other navbar elements at smaller screen sizes). Close #230

### CODE REVIEW

File: projects/developer/src/app/navbar/navbar.component.html [hunk 1]
```diff
@@ -27,7 +30,7 @@
             </div>
             <p-slideMenu #menu [model]="itemsMobile" [popup]="true" [viewportHeight]="340"></p-slideMenu>
             <button pButton type="button" class="navbar__mobile-nav-button" icon="fa fa-bars"
-                    (click)="menu.toggle($event)"></button>
+                (click)="menu.toggle($event)"></button>
```
Comment (mheppner) [inline projects/developer/src/app/navbar/navbar.component.html:33]: Although these are good changes, fixing linting/styling clutters up PRs. I would prefer to see separate PRs for just those. A PR should have a clear description of what is being changed and the diff should clearly match the description.

File: projects/developer/src/app/navbar/navbar.component.scss [hunk 2]
```diff
@@ -74,7 +73,7 @@ $navbar-item-width: 150px;
 
 .navbar__logo {
     color: var(--white);
-    margin-right: 1.0em;
+    margin-right: 1em;
```
Comment (mheppner) [inline projects/developer/src/app/navbar/navbar.component.scss:76]: Same with this file, a big diff of style changes, when only two style rules were added.

File: scale7.code-workspace [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (mheppner) [inline scale7.code-workspace:1]: I'm not sure what this file is?

**Comment by Nikkala-AIS** [comment]
It was decided that this feature is unnecessary.

**Pull request closed without merging**